app/views/events/_table_badges.html.erb

```diff
@@ -1,30 +1,253 @@
+            <h4 class="modal-title">No Society Credit</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario is not being run for organized play credit. You will not be receiving a chronicle for
+              playing. It may be pregen only, with Society characters, or otherwise. See "More Info!" for more
+              details.</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
 <% end %>
 <% if table.raffle? %>
-    <span class="badge badge-raffle">Raffle</span>
+    <button type="button" class="btn btn-round-xs btn-raffle btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_raffle">Raffle</button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_raffle" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Raffle</h4>
+          </div>
+          <div class="modal-body">
+            <p>This table will be offered as a raffle during the event. You will need purchase raffle tickets, which
+              will be drawn prior to the session. See "More Info!" for mode details on when the raffle will happen, and
+              for the raffle ticket price.</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
 <% end %>
+
+
 <% if table.premium? %>
-    <span class="badge badge-premium">Premium</span>
-    <span class="badge badge-cost"><%= number_to_currency table.price %></span>
+    <button type="button" class="btn btn-round-xs btn-premium btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_premium">Premium</button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_premium" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Premium Table</h4>
+          </div>
+          <div class="modal-body">
+            <p>This is a premium table, with a VIP Game Master!
+              This table will cost an additional <%= number_to_currency table.price %> to play
+              at, with the proceeds going to our charity.
+              <% if table.early_bird_discount? %>
+                  After <%= table.prereg_ends.localtime.to_formatted_s(:short) %> the price will go up
+                  to  <%= number_to_currency table.onsite_price %>.
+              <% end %>
+            </p>
```

Comment:
Reword: "... with proceeds going to Skalcon's charity". "... the price will rise to ..."